website/integrations/services/apple/index.md

```diff
@@ -0,0 +1,295 @@
+first_name, _, last_name = request.user.name.partition(" ")
+
+return {
+  "name": request.user.name,
+  "given_name": first_name,
+  "family_name": last_name,
+  "preferred_username": request.user.username,
+  "nickname": request.user.username,
+  "groups": [group.name for group in request.user.ak_groups.all()],
+}
+```
+
+Save the scope mapping and press "Create" to continue to the next scope mapping.
+
+#### Read Scope
+
+Next, create a new scope mapping to handle read access to the user's profile information:
+
+| Field      | Value                           |
+| ---------- | ------------------------------- |
+| Name       | Apple Business Manager ssf.read |
+| Scope name | ssf.read                        |
+| Expression | `return {}`                     |
+
+Once again, save the scope mapping and press "Create" to continue to create the final scope mapping.
+
+#### Manage Scope
+
+| Field      | Value                             |
+| ---------- | --------------------------------- |
+| Name       | Apple Business Manager ssf.manage |
+| Scope name | ssf.manage                        |
+| Expression | `return {}`                       |
+
+Finally, save the scope mapping and visually confirm the creation of all three scope mappings.
+
+### OAuth2/OpenID Provider
+
+From the authentik Admin interface, navigate to **Applications** -> **Providers** and press **Create**.
+
+:::tip{title="Keep your text editor ready"}
+
+authentik will automatically generate the _Client ID_ and _Client Secret_ values for the new provider.
+
+You'll need these values when configuring Apple Business Manager.
+
+:::
+
+From the creation wizard, select **OAuth2/OpenID Provider** and continue with the following form values:
+
+| Field                 | Value                                               |
+| --------------------- | --------------------------------------------------- |
+| Name                  | Apple Business Manager                              |
+| Authorization Flow    | `default-provider-authorization-implicit-consent`   |
+| Redirect URIs/Origins | `https://gsa-ws.apple.com/grandslam/GsService2/acs` |
+
+Under **Advanced protocol settings**, make sure to add the newly created scope mappings to the provider, as well as the following additional scopes:
+
+1. `ssf.manage`
+2. `ssf.read`
+3. `profile`
+4. `authentik default OAuth Mapping: OpenID 'email'`
+5. `authentik default OAuth Mapping: OpenID 'openid'`
+6. `authentik default OAuth Mapping: OpenID 'offline_access'`
+
+Visually confirm that you've copied the **Client ID** and **Client Secret** before pressing **Finish**. You'll need these values when configuring Apple Business Manager. Save the provider and confirm that the new provider is listed in the provider overview.
+
+:::tip{title="OpenID Configuration URL"}
+
+Before leaving the provider overview page, make sure to copy the **OpenID Configuration URL** value to your text editor. You'll need this when configuring Apple Business Manager.
```

Comment:
```suggestion
Before leaving the provider overview page, make sure to copy the `OpenID Configuration URL` value to your text editor. You'll need this when configuring Apple Business Manager.
```

Reply:
This one could go either way... if you were referring to the field label itself, the UI component, then bold it. BUt since you are talking about the alue related to that field, it's a value, so code font it is.